xml/System.Windows.Forms/GiveFeedbackEventArgs.xml

```diff
@@ -322,7 +322,11 @@
         <summary>Gets or sets a value indicating whether a layered window drag image is used.</summary>
         <value>
           <see langword="true" /> for <see cref="P:System.Windows.Forms.GiveFeedbackEventArgs.UseDefaultDragImage" /> to use a layered window drag image with a size of 96x96; otherwise <see langword="false" />.</value>
-        <remarks>To be added.</remarks>
+        <remarks>
+          <para>
+            Specify <see langword="true" /> for <see cref="P:System.Windows.Forms.GiveFeedbackEventArgs.UseDefaultDragImage" /> to use a layered window drag image with a size of 96x96; otherwise <see langword="false" />.
+          </para>
+        </remarks>
```

Comment:
```suggestion
        <remarks>Specify <see langword="true" /> for <see cref="P:System.Windows.Forms.GiveFeedbackEventArgs.UseDefaultDragImage" /> to use a layered window drag image with a size of 96x96; otherwise <see langword="false" />.</remarks>
```